packages/react-native-renderer/src/ReactNativeFrameScheduling.js

```diff
@@ -49,12 +49,13 @@ function scheduleDeferredCallback(
 ): number {
   // We assume only one callback is scheduled at a time b'c that's how Fiber works.
   scheduledCallback = callback;
-  return setTimeout(setTimeoutCallback, 1);
+  const timeoutId = setTimeout(setTimeoutCallback, 1);
+  return (timeoutId: any); // Timeouts are always numbers on RN
```

Comment:
I could keep an object mapping but doesn't seem worth it in this case.
Flow now considers timeout to be a special type (which is good for environments like Node where it's not a number).